src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/ViewRenderer.js

```diff
@@ -48,37 +49,36 @@ class ViewRenderer extends React.Component<Props> {
         }
     }
 
-    getView = (route: Route): View => {
-        const View = viewRegistry.get(route.type);
-
-        if (!View) {
-            throw new Error('View "' + route.type + '" has not been found');
-        }
-
-        return View;
-    };
-
     renderView(route: Route, child: Element<*> | null = null) {
         const {router} = this.props;
-        const View = this.getView(route);
+        const CurrentView = viewRegistry.get(route.type);
+        const viewConfig = viewRegistry.getConfig(route.type);
 
         let viewKey = getViewKeyFromRoute(route, router.attributes) || '';
-        if (View.remountViewOnLogin) {
+        if (CurrentView.remountViewOnLogin) {
             viewKey = viewKey + '__' + this.loginCount;
         }
 
         const element = (
-            <View
+            <CurrentView
                 isRootView={!route.parent}
                 key={viewKey}
                 route={route}
                 router={router}
             >
                 {(props) => child ? React.cloneElement(child, props) : null}
-            </View>
+            </CurrentView>
         );
 
         if (!route.parent) {
+            if (!viewConfig.rootSpaceless) {
```

Comment:
The new configuration was required. There actually was no way to get it work without it specially in cases of nested tab views (version & activities) and Tabs inside WebspaceTabs. Funfact actually with this new option there is no bc break and so custom views still get the same padding like they did before.